tests/language/test_language_service.py

```diff
@@ -188,7 +190,9 @@ def test_language_flavor(self):
         pgsql_params = LanguageFlavorChangeParams.from_data('file://pguri.sql', 'sql', 'pgsql')
         mssqql_params = LanguageFlavorChangeParams.from_data('file://msuri.sql', 'sql', 'mssql')
         other_params = LanguageFlavorChangeParams.from_data('file://other.doc', 'doc', '')
```

Comment:
What would happen if we add a mysql_params here? As we made changes to that logic, we should add tests to cover it as well. You can add them as part of you completion test changes.